cmd/otelcol/config/collector/ecs_ec2_config.yaml

```diff
@@ -0,0 +1,135 @@
+# This collector config file is designed for use within an ECS task.
+# The collector should run in a sidecar container within an ECS task.
+config_sources:
+  env:
+    defaults:
+      METRICS_TO_EXCLUDE: []
+      ECS_METADATA_EXCLUDED_IMAGES: []
+
+extensions:
+  health_check:
+    endpoint: 0.0.0.0:13133
+  http_forwarder:
+    ingress:
+      endpoint: 0.0.0.0:6060
+    egress:
+      endpoint: "https://api.${SPLUNK_REALM}.signalfx.com"
+  zpages:
+    endpoint: 0.0.0.0:55679
+
+receivers:
+  jaeger:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:14250
+      thrift_http:
+        endpoint: 0.0.0.0:14268
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+      http:
+        endpoint: 0.0.0.0:55681
```

Comment:
Is this needed? Not sure this is enabled in any other config by default

Reply:
Not sure if needed but this http endpoint is enabled in all default configs!

Reply:
The port changes to 4318 in the latest release not sure if/when we should switch to that port

Reply:
Living the port the same for now.